src/Shaders/ShadersInclude/pbrBlockSubSurface.fx

```diff
@@ -195,6 +195,10 @@ struct subSurfaceOutParams
             float refractionLOD = getLodFromAlphaG(vRefractionMicrosurfaceInfos.x, alphaG);
         #endif
 
+        // Scale refractionLOD with IOR so that an IOR of 1.0 results in no microfacet refraction and
+        // an IOR of 1.5 results in the default amount of microfacet refraction.
+        refractionLOD = mix(refractionLOD, 0.0, clamp(vRefractionInfos.y * 3.0 - 2.0, 0.0, 1.0));

```

Comment:
I don't understand this correction.

`refractionLOD` is the mipmap level to use to sample the `refractionSampler`, why would it be related to IOR? In my understanding, `refractionLOD` should only be related to the alphaG/roughness of the material and IOR is used to compute the refraction vector.

Reply:
The blurred light transmission is caused by micro-facet refractions that we model as roughness. As the IOR gets close to 1, the amount of refraction decreases to nothing (assuming the surrounding medium is air). A piece of rough diamond "blurs" the light transmitted through it more than a piece of glass of the same roughness.
Now, that said, my approximation is a simple linear scale which may not be as correct as it can be. The previous micro-refraction amount was already very approximate, of course.

Reply:
Ok, so you make a link between blurriness and IOR: the higher the IOR, the higher the blurriness. Is that really true? Shouldn't we instead use a new material property to model this blurriness? It seems in Arnold they name that "Extra roughness": https://docs.arnoldrenderer.com/display/A5AFMUG/Transmission#Transmission-ExtraRoughness (they have also other parameters that seem to impact the bluriness, like "dispersion abbe" or "transmission scatter").

Also, I'm concerned that it is a breaking change, so we should be careful and make sure it was a bug and/or add a switch to enable/disable this feature. @sebavan thoughts?

Reply:
It's really true, and the effect is reproduced in various path tracers including Enterprise PBR tracers and Blender Cycles.  The blurriness comes from not just the intensity of the microfacet roughness, but also the amount by which those microfacets are capable of distorting the light.  In the case of IOR 1.0, the light is able to cross the boundary at any angle and not change its vector at all, so the microfacets have no capability to produce any blurriness at all in the transmission, only on the specular reflection.

I made a test model:  [TransmissionRoughnessTest.zip](https://github.com/BabylonJS/Babylon.js/files/5958739/TransmissionRoughnessTest.zip)

If you import this into Blender though, you'll have to manually reassign the IOR values for about 5 materials.

Reply:
Ok, thanks for the explanation, will let @sebavan answer about this one.

Reply:
Yup love it I d consider it a fix but the linear approach at the LOD level sounds weird knowing our LODs are not stored linearly. I wondered if it should be applied to alphaG instead or better at the perceptual roughness level which is the only linear value ?

Reply:
@MiiBond Do you know?

Reply:
Yes, that might be more correct. I'll look into this more this week.

Reply:
Okay, I updated my logic to scale the roughness or alpha depending on the IOR. Results look closer to what Dimension renders for high roughness with low IOR. Of course, the screen space refraction is kind of hard to compare to a pathtracer :)

Reply:
@sebavan when you guys get a chance, this could be checked again.

Reply:
Looks pretty good to me, @Popov72 could you do a last check so that we can merge it ?

@bghgary any issues you foresee with merging ?

Reply:
LGTM